rllib/algorithms/marwil/marwil.py

```diff
@@ -120,14 +122,7 @@ def __init__(self, algo_class=None):
             "is": {"type": ImportanceSampling},
             "wis": {"type": WeightedImportanceSampling},
         }
-        deprecation_warning(
-            old="MARWIL currently uses off_policy_estimation_methods: "
```

Comment:
I think @Rohan138 introduces this a couple of days ago. I may be wrong here, but if so, this would be a hard deprecation.